Ur-Ghast, Platinum, Firestone, & Sulfur Bee Changes

UltraProdigy opened the pull request:
## Ur-Ghast Bee Change:
This changes the Ur-Ghast bee block base requirement to the Gregtech thaumium block rather than the thaumcraft one. This will be consistent with the Thaumium Dust bee which already had that.

This is based on and will close https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/20015

### Pre-Change:
![image](https://github.com/user-attachments/assets/6d7b7b3a-4893-4719-bb54-8028f8f28d03)

### Post-Change:
![image](https://github.com/user-attachments/assets/47131cd8-e289-48a2-b491-06e00f985641)

## Platinum Bee Change:
This changes the Platinum bee block base requirement from the nickel block back to platinum; it had no reason to be the nickel block.

This is based on and will close https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17761

### Pre-Change:
![image](https://github.com/user-attachments/assets/b6da3b5a-71a4-4db3-a892-d78c46eb69b1)

### Post Change:
![image](https://github.com/user-attachments/assets/9699e4db-96c8-4065-9f2f-e43398648088)

## Sulfur Bee Change:
Currently, the Sulfur bee drops the firestone comb as it's specialty, at the same rate as the firestone bee itself, thereby rendering the firestone bee useless. The specialty output has now been changed to a new "Pyrite Comb", which processes in the LCR with HCl to get the it's purified ore variant. This change will allow both the firestone bee to be(e) useful, and the give the sulfur bee an enhanced purpose (& also be consistent with it's in-game ore-vein family).

This is based on and will close https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/19816

### Pre-Change:
![image](https://github.com/user-attachments/assets/49ff053b-12d6-4e2d-94b2-a704d77b8bfe)
![image](https://github.com/user-attachments/assets/274b96af-bfe2-4139-9869-9733ec03739b)

###  Post Change:
![image](https://github.com/user-attachments/assets/e19ff81d-57c8-42fd-aef3-e3f7c35b76b3)
![image](https://github.com/user-attachments/assets/d8b91934-83f3-4bf0-a91d-0b7383353d1e)

LazyFleshWasTaken commented:
I believe sulfur bee was given firestone comb for a reason, and removing it outright is a fairly big change. I recommend leaving it in a specialty, along with pyrite, but giving it a lower chance, i.e. 5% or 2.5%

This still gives a use to the firestone bee, especially with the non-ngeligible amount of firestone used in later game.

UltraProdigy commented:
Good call, GoG still needs it. I'll put it back in, and maybe increase the chance a little (maybe 20% from 15%) on the Firestone bee to incentivize its usage if that sounds like a good idea?

UltraProdigy commented:
## New Changes:
- Reverted Sulfur bee to have 5% specialty firestone comb output to account for GoG players.
- Increased Firestone bee to have a 20% specialty firestone comb output to incentivize bee usage over sulfur.
- Changed the color of the Sulfur bee to match it's comb... the only reason it was blue before is because it's code was copy pasted from the Electrotine bee above it, hence why their colors coincidentally match perfectly.

### Pre-Change:
![image](https://github.com/user-attachments/assets/ccad22df-8f27-4873-899c-a5c2bc0db969)

### Post-Change:
![image](https://github.com/user-attachments/assets/5514b8a5-7223-4bdf-87a7-4a668b85b75f)
![image](https://github.com/user-attachments/assets/b1fb5413-22d6-44e2-a6fc-9a737fbe55b8)
